examples/bert/build.py

```diff
@@ -101,17 +103,91 @@ def parse_arguments():
                             'RobertaForQuestionAnswering',
                             'RobertaForSequenceClassification',
                         ])
+    parser.add_argument('--model_dir', type=str, required=True)
```

Comment:
I suggest we set the required to False here for backward compatibilty or we need to add 
`        state_dict = torch.load(os.path.join(args.model_dir, "pytorch_model.bin"))
        hf_bert.load_state_dict(state_dict, strict=False)
` for all model type, right?

Reply:
Yes that makes sense. Making `--model_dir` optional and defaults to previous behavior.